ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLProvider.java

```diff
@@ -31,14 +33,20 @@
 
 public class GraphQLProvider {
 
+  public static final int MAX_COMPLEXITY = 200;
```

Comment:
Is this to be configurable?

Reply:
No.  Normal queries won't break that limit.